### PR TITLE
Remove extra String.includes helper

### PR DESCRIPTION
**Pull request by choumx**
Stumbled across this. We already have a polyfill installed on the prototype in array-includes.js. 

/to @caroqliu @samouri /cc @jridgewell

### CODE REVIEW

File: build-system/test-configs/conformance-config.textproto [hunk 1]
```diff
@@ -68,12 +68,6 @@ requirement: {
   value: 'string.prototype.endsWith'
 }
 
-requirement: {
-  type: BANNED_PROPERTY_CALL
-  error_message: 'string.prototype.includes is not allowed'
-  value: 'string.prototype.includes'
-}
```
Comment (choumx) [inline build-system/test-configs/conformance-config.textproto]: This has clearly not been very consistent.

**Comment by samouri** [comment]
I'm confused. `array-includes` polyfill adds support for Array.prototype.includes whereas in this PR I think you are addressing `String.prototype.includes`?

**Comment by choumx** [comment]
Oops, you're right. 🙃 I thought we had a prototype polyfill for String.includes.

**Comment by choumx** [comment]
Closing. Separately we can consider adding an analogous proto polyfill for String.includes to reduce confusion.

**Pull request closed without merging**